botocore/client.py

```diff
@@ -523,9 +526,15 @@ class Config(object):
         * User agent extra
 
     """
+
+    DEFAULT_TIMEOUT = 60
```

Comment:
We should probably not have this defined twice (here and in endpoints.py).  Let's consolidate this so the default timeout value lives in a single place.